jobs/looker-utils/looker_utils/main.py

```diff
@@ -69,6 +69,9 @@ def delete_branches(ctx, project, inactive_days, exclude):
         days=inactive_days
     )
 
+    # switch to dev mode
+    sdk.update_session({"workspace_id":"dev"})
```

Comment:
The [source for this method](https://github.com/looker-open-source/sdk-codegen/blob/5223398b5e1cf960bc8cc6130906e133b08e990b/python/tests/integration/test_methods.py#L277) uses a `WriteApiSession` object instead of a dict so I think that would be more correct but maybe that's just a wrapper around a dict and this works as well